### PR TITLE
DOCS README add repo structure section

### PR DESCRIPTION
**Pull request by KoenBal**
### Description
The repo structure section gives potential contributors an immediate idea of what parts they could contribute to without the need to having to dive into the folders.

Add a repo structure to the main README file.
See #1976 for some questions I have that could improve this section.

part of a larger README update

### CODE REVIEW

**Comment by timmoreton** [comment]
Thanks for the contributions! I’ll get chance to take a look later this week

**Comment by KoenBal** [comment]
> Thanks for the contributions! I’ll get chance to take a look later this week

No worries. I already assumed there is more pressing work to do now with the upcoming launch. A great engineer strives to focus ;).

File: README.md [hunk 1]
```diff
@@ -35,6 +35,32 @@ See [Developer's Guide](https://docs.celo.org/) for full details about the desig
 
 See the [issue backlog](https://github.com/celo-org/celo-monorepo/issues) for a list of active or proposed tasks. Feel free to create new issues to report bugs and/or request features. Please add labels to your issues, tagging the appropriate package/area.
 
+### 📂 Repo structure
+The repository has the following packages (sub projects):
+- [analytics](packages/analytics) - Cloud Dataflow/Apache Beam jobs
```
Comment (timmoreton) [inline README.md]: add `for processing Celo Wallet logs and telemetry`

File: README.md [hunk 2]
```diff
@@ -35,6 +35,32 @@ See [Developer's Guide](https://docs.celo.org/) for full details about the desig
...
+- [attestation-service](packages/attestation-service) - service run by validators on the Celo network to send SMS messages, enabling attestations of user phone numbers and their accounts on the Celo network
+- [blockchain-api](packages/analytics) - example of how to compile TypeScript files while deploying to App Engine (correct?)
```
Comment (timmoreton) [inline README.md]: `Service that uses Blockscout to present view of transactions by account for Celo Wallet activity feed`

File: README.md [hunk 3]
```diff
@@ -35,6 +35,32 @@ See [Developer's Guide](https://docs.celo.org/) for full details about the desig
...
+- [celotool](packages/celotool) - tool for various scripts that the engineering team might run
```
Comment (timmoreton) [inline README.md]: `Scripts for deploying and managing testnets`

File: README.md [hunk 4]
```diff
@@ -35,6 +35,32 @@ See [Developer's Guide](https://docs.celo.org/) for full details about the desig
...
+- [cli](packages/cli) - tool for interacting with the Celo protocol ([docs](https://docs.celo.org/getting-started/using-the-cli))
```
Comment (timmoreton) [inline README.md]: tool that uses ContractKit to..

File: README.md [hunk 5]
```diff
@@ -35,6 +35,32 @@ See [Developer's Guide](https://docs.celo.org/) for full details about the desig
...
+- [contractkit](packages/contractkit) - library to help developers and validators interact with the celo-blockchain ([docs](https://docs.celo.org/celo-sdk/contractkit))
```
Comment (timmoreton) [inline README.md]: maybe `with the protocol`` or `with the smart contracts`

File: README.md [hunk 6]
```diff
@@ -35,6 +35,32 @@ See [Developer's Guide](https://docs.celo.org/) for full details about the desig
...
+- [dappkit](packages/dappkit) - set of functions for mobile DApps to work with the wallet app (ex. sign transactions and access the user's account) ([docs](https://docs.celo.org/celo-sdk/dappkit))
+- [dev-utils](packages/dev-utils) - no README available (improve?)
```
Comment (timmoreton) [inline README.md]: A `utils` package for use as a devDependency

File: README.md [hunk 7]
```diff
@@ -35,6 +35,32 @@ See [Developer's Guide](https://docs.celo.org/) for full details about the desig
...
+- [docs](packages/docs) - technical documentation for the celo project ([live](https://docs.celo.org/))
+- [faucet](packages/faucet) - faucet deployment configuration ([live](https://celo.org/build/faucet))
+- [helm-charts](packages/helm-charts) - templatized deployments of entire environments to Kubernetes clusters (improve?)
+- [mobile](packages/mobile) - Android wallet app for the celo platform ([docs](https://docs.celo.org/getting-started/using-the-mobile-wallet), [live](https://play.google.com/store/apps/details?id=org.celo.mobile.alfajores))
+- [notification-service](packages/notification-service) - service for dispatching notifications to mobile clients
```
Comment (timmoreton) [inline README.md]: managing push notifications for Celo Wallet

File: README.md [hunk 8]
```diff
@@ -35,6 +35,32 @@ See [Developer's Guide](https://docs.celo.org/) for full details about the desig
...
+- [protocol](packages/protocol) - identity, stability and other smart contracts for the celo protocol ([docs](https://docs.celo.org/celo-codebase/protocol))
```
Comment (timmoreton) [inline README.md]: nit: celo -> `Celo`

File: README.md [hunk 9]
```diff
@@ -35,6 +35,32 @@ See [Developer's Guide](https://docs.celo.org/) for full details about the desig
...
+- [react-components](packages/react-components) - no README available (improve?)
+- [terraform-modules](packages/terraform-modules) - tool that allows developers to treat infrastructure as code
```
Comment (timmoreton) [inline README.md]: `templatized deployments of entire VM-based testnets for Google Cloud `

File: README.md [hunk 10]
```diff
@@ -35,6 +35,32 @@ See [Developer's Guide](https://docs.celo.org/) for full details about the desig
...
+- [transaction-metrics-exporter](packages/transaction-metrics-exporter) - tool to export celo transaction metrics (improve?)
```
Comment (timmoreton) [inline README.md]: `Monitoring tool that executes transactions on the network and exports testnet-level prometheus metrics`

File: README.md [hunk 11]
```diff
@@ -35,6 +35,32 @@ See [Developer's Guide](https://docs.celo.org/) for full details about the desig
...
+- [typescript](packages/typescript) - no README available (improve?)
+- [utils](packages/utils) - no README available (improve?)
+- [verification-pool-api](packages/verification-pool-api) - firebase cloud functions to process verification requests
```
Comment (timmoreton) [inline README.md]: (deprecated) A service that handles a pool of Verifier App instances and requests them to direct SMS for attestation purposes

File: README.md [hunk 12]
```diff
@@ -35,6 +35,32 @@ See [Developer's Guide](https://docs.celo.org/) for full details about the desig
...
+- [verifier](packages/verifier) - Android verifier app to send SMS messages, enabling attestations of user phone numbers and their accounts on the Celo network (correct, improve?)
+- [walletkit](packages/walletkit) - no README available (improve?)
```
Comment (timmoreton) [inline README.md]: `(deprecated) common functions to access smart contracts used by Celo Wallet`

File: README.md [hunk 13]
```diff
@@ -35,6 +35,32 @@ See [Developer's Guide](https://docs.celo.org/) for full details about the desig
...
+- [notification-service](packages/notification-service) - service for dispatching notifications to mobile clients
+- [protocol](packages/protocol) - identity, stability and other smart contracts for the celo protocol ([docs](https://docs.celo.org/celo-codebase/protocol))
+- [react-components](packages/react-components) - no README available (improve?)
+- [terraform-modules](packages/terraform-modules) - tool that allows developers to treat infrastructure as code
+- [transaction-metrics-exporter](packages/transaction-metrics-exporter) - tool to export celo transaction metrics (improve?)
+- [typescript](packages/typescript) - no README available (improve?)
+- [utils](packages/utils) - no README available (improve?)
+- [verification-pool-api](packages/verification-pool-api) - firebase cloud functions to process verification requests
+- [verifier](packages/verifier) - Android verifier app to send SMS messages, enabling attestations of user phone numbers and their accounts on the Celo network (correct, improve?)
```
Comment (timmoreton) [inline README.md]: correct!

**Review by timmoreton — COMMENTED** [review]
Thanks for the contribution! Should have had this a long time ago. PTAL at the comments and see what else you can fill in.

**Comment by KoenBal** [comment]
Thx for the feedback.
Pushed adjustments.
@timmoreton could un-assign me but not assing it back to you.
#2111 Should solve all these assigning problems.

**Comment by timmoreton** [comment]
Wrapped these up in #2201 - PTAL!

**Pull request closed without merging**